edit-member.html

```diff
@@ -189,11 +189,26 @@ <h1>Edit Profile</h1>
             </form>
           </div> <!-- tab-pane.// -->
           <div class="tab-pane fade" id="nav-tab-paypal">
-          <p>To be updated soon</p>
-          
+            <div>
+              <div class="form-group">
+                <label for="InputPassword">Current Password</label>
+                <input type="Password" class="form-control" id="InputEmail1" aria-describedby="emailHelp" placeholder="currentPassword">
+              </div>
+              <div class="form-group">
+                <label for="InputPassword1">New Password</label>
+                <input type="password" class="form-control" id="InputPassword1" placeholder="New Password">
+              </div>
+              <div class="form-group">
+                <label for="InputPassword1">Repeat New Password</label>
```

Comment:
Can you update the label's `for` attribute? It has to be consistent with the id of the password you provided. which in this case is `InputPassword2`